src/utils.js

```diff
@@ -40,6 +41,21 @@ export function chainWith(createDependentObservable) {
   });
 }
 
+/**
+ * Custom rxjs operator to emit an initial value even if each observable hasn't emmited a value yet .
```

Comment:
Custom rxjs operator that works like combineLatest, but emits even if some of the source observables haven't emitted yet.


https://www.learnrxjs.io/learn-rxjs/operators/combination/combinelatest